ecs-init/exec/iptables/iptables.go

```diff
@@ -95,11 +105,18 @@ func (route *NetfilterRoute) Remove() error {
 		preroutingErr = fmt.Errorf("error removing prerouting chain entry: %v", preroutingErr)
 	}
 
-	var inputErr error
+	var localhostInputError, introspectionInputError error
```

Comment:
nit: move `introspectionInputError` closer to its scope in line 115

Reply:
these two are used in line 129